### ISSUE TITLE
we don't show timing in the main log output

### ISSUE THREAD

**Issue by yegor256** [objectionary/eoc#368]
Currently, I'm getting this:

```
$ eoc compile
EO objects registered in .eoc/eo-foreign.json
EO program assembled in .eoc
EO program verified in .eoc
Java sources generated in .eoc/generated-sources
Java .class files compiled into .eoc
```

Would be much better to see this:

```
EO objects registered in .eoc/eo-foreign.json in 34ms
EO program assembled in .eoc in 1s
EO program verified in .eoc in 340ms
Java sources generated in .eoc/generated-sources in 5s
Java .class files compiled into .eoc in 2s
```

Let's add the timings at the end of every logging line.

**Comment by ReBy298** [comment]
I would like to contribute to this issue :D

**Comment by yegor256** [comment]
@ReBy298 please, go head!

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#434](https://github.com/objectionary/eoc/issues/434), [#435](https://github.com/objectionary/eoc/issues/435) are still not solved.

**Comment by 0pdd** [comment]
@yegor256 all 2 puzzles are solved here: [#434](https://github.com/objectionary/eoc/issues/434), [#435](https://github.com/objectionary/eoc/issues/435).
